### ISSUE TITLE
Searchbar on the apppage

### ISSUE THREAD

**Issue by Feichtmeier** [ubuntu-flutter-community/software#855]
As a result of the meeting:

- make the search available on the AppPage

@d-loose could you eventually tackle this one?

**Comment by madsrh** [comment]
> ... a suggestion dropdown for this or maybe a change from "search as you type" to "search on enter" is enough?

If a drop-down is possible that would be very slick 😎

**Comment by Feichtmeier** [comment]
I think we need to discuss this again. The best would be if you would be there if we discuss this with @anasereijo and @madsrh 

Another way would be to move the search logic out of exploremodel one level higher in the AppModel and only pass the list of AppFindings to the lower pages if searched

**Comment by anasereijo** [comment]
Agree. This needs a bit more thinking and discussion. We can talk about it on next week's 'work together' session? @d-loose I can add you to it if you are free! (Monday 5pm - UK)

**Comment by d-loose** [comment]
@anasereijo can't promise I'll be there very long, but I'll try to join for half an hour or so.
